CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Removed
 ### Fixed
 
+## [7.2.0] - 2021-11-16
+### Changed
+- `templates/__common__/config/tasks/graphics-meta.js` - check for `data-graphic` and `data-feature`, set the type of each template (graphic or feature) in the metadata
+- `templates/__common__/utils/deployment/update-log-sheet.js` - get the type set in `graphics-meta` to determine the tab in the log sheet to paste metadata to, write metadata to the respective tab unless the hostname is `capybara-test`
+
```

Comment:
maybe this comes later in the process, but I noticed there are some other files changed for 🧀 queso classes 🧀